app/controllers/MobileController.scala

```diff
@@ -0,0 +1,91 @@
+package controllers
+
+import java.sql.Timestamp
+import java.time.Instant
+import java.util.UUID
+import javax.inject.Inject
+
+import com.mohiva.play.silhouette.api.{Environment, Silhouette}
+import com.mohiva.play.silhouette.impl.authenticators.SessionAuthenticator
+import com.vividsolutions.jts.geom._
```

Comment:
I don't think a lot of these imports are necessary. Could you go through and remove the ones that we don't need? And if you copied these from somewhere and the imports aren't used there either, def feel free to delete the unused imports there as well!